Glyssen/Dialogs/NarrationOptionsDlg.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Glyssen.Bundle;
+
+namespace Glyssen
+{
+	public partial class NarrationOptionsDlg : Form
+	{
+		private const int kInvalidNarratorNum = -1;
+		private int m_intNarratorNum = kInvalidNarratorNum;
+		private readonly Project m_project;
+
+		public NarrationOptionsDlg(Project project)
+		{
+			InitializeComponent();
+			m_project = project;
+
+			if (m_project.ProjectSettings.NumberOfNarrators != ProjectSettings.kNumberOfNarratorsNotSet)
+			{
+				m_intNarratorNum = m_project.ProjectSettings.NumberOfNarrators;
```

Comment:
If this has not been set before, I think we want to automatically set it to a valid default value that makes sense. Not sure if it should be 1 (which would mimic the current behavior) or a value calculated based on some heuristic that reflects the books included in the project.